Yafc.UI/ImGui/ImGuiBuilding.cs

```diff
@@ -141,6 +142,7 @@ public void DrawText(Rect rect, string text, RectAlignment alignment = RectAlign
         var fontSize = GetFontSize(font);
         var cache = textCache.GetCached((fontSize, text, uint.MaxValue));
         var realRect = AlignRect(rect, alignment, cache.texRect.w / pixelsPerUnit, cache.texRect.h / pixelsPerUnit);
```

Comment:
few more vars

Reply:
The `var`s that were replaced, were replaced only by Code Cleanup. I did not go after vars in this PR by myself.